spec/workers/digest_run_worker_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe DigestRunWorker do
+  describe ".perform" do
+    shared_examples "tests for critical and warning states" do
+      context "find digest runs from last  minutes" do
```

Comment:
Just a minor thing but there's an extra space here, or did you mean to say `last x minutes`?

Reply:
Thanks @deborahchua, have reworded